test/integration/requests.test.ts

```diff
@@ -256,4 +274,125 @@ describe("/requests", function () {
         });
     });
   });
+
+  describe("GET /requests", function () {
+    it("should return all requests", function (done) {
+      chai
+        .request(app)
+        .get("/requests?dev=true")
+        .end(function (err, res) {
+          expect(res).to.have.status(200);
+          expect(res.body.data).to.have.lengthOf(2);
+          expect(res.body.data[0]).to.have.all.keys([
+            "id",
+            "createdAt",
+            "requestedBy",
+            "from",
+            "until",
+            "lastModifiedBy",
+            "message",
+            "updatedAt",
+            "state",
+            "type",
+          ]);
+          done();
+        });
+    });
+
+    it("should return all requests by specific user", function (done) {
+      chai
+        .request(app)
+        .get("/requests?dev=true&requestedBy=testUser")
+        .end(function (err, res) {
+          expect(res).to.have.status(200);
+          expect(res.body.data.every((request: any) => request.requestedBy === "testUser"));
+          done();
+        });
+    });
+
```

Comment:
the `expect` statement is missing a condition to evaluate whether every request's `requestedBy` field equals `testUser`.

Reply:
No, she is checking every object that has same requestedBy id or not